bundlesize.config.json

```diff
@@ -26,11 +34,11 @@
     },
     {
       "path": "packages/react-instantsearch-hooks/dist/umd/ReactInstantSearchHooks.min.js",
-      "maxSize": "45.20 kB"
+      "maxSize": "46.25 kB"
     },
     {
       "path": "packages/react-instantsearch-hooks-web/dist/umd/ReactInstantSearchHooksDOM.min.js",
-      "maxSize": "55.50 kB"
+      "maxSize": "56 kB"
     },
```

Comment:
what changed this, the 3.13.3 update?

Reply:
I can't say for sure. Bundlephobia doesn't report that much of a bump (and there's a delta in the base size as well, but it might be because of how it's calculated on their end): https://bundlephobia.com/scan-results?packages=algoliasearch-helper@3.13.3,algoliasearch-helper@3.13.2.